src/main/java/de/samply/share/broker/rest/Monitoring.java

```diff
@@ -243,6 +242,7 @@ public Response createQuery(@HeaderParam(HttpHeaders.AUTHORIZATION) String auth,
    * @param queryContainer the query as cql string and the target sites as list
    * @return response if the query has been created
    */
+  @Secured
```

Comment:
Please use a separate annotation for the monitoring account (separate filter) here.